external/fv3viz/examples/plot_cross.py

```diff
@@ -0,0 +1,21 @@
+"""
+"Cross" plot
+==========================
+
+The routine ``vcm.cubedsphere.to_cross`` is useful for quick plots when
+cartopy may not be installed.
```

Comment:
Having to load and merge the grid coordinates from another datasource is often a pain too:
```suggestion
cartopy may not be installed and/or latitude and longitude coordinates
are not easily available.
```